src/main/java/appeng/me/cache/GridStorageCache.java

```diff
@@ -329,4 +356,78 @@ public void applyChanges() {
             }
         }
     }
+
+    private void updateBytesInfo() {
+        this.itemBytesTotal = 0;
+        this.itemBytesUsed = 0;
+        this.fluidBytesTotal = 0;
+        this.fluidBytesUsed = 0;
+        this.essentiaBytesTotal = 0;
+        this.essentiaBytesUsed = 0;
+        try {
+            for (ICellProvider icp : this.activeCellProviders) {
+                if (icp instanceof TileDrive) {
+                    System.out.println();
```

Comment:
What is the purpose of this line?

Reply:
damn.... why i always forget something..